bundle/edu.gemini.phase2.skeleton.servlet/skeleton.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
```

Comment:
I thought `skeleton.sh` was present somewhere on the codebase (It may have been on the svn repo), but in any case great to have it here